apps/web/playwright/managed-event-types.e2e.ts

```diff
@@ -216,3 +251,10 @@ async function gotoBookingPage(page: Page) {
 
   await page.goto(previewLink ?? "");
 }
+
+async function submitAndWaitForResponse(page: Page) {
+  const submitPromise = page.waitForResponse("/api/trpc/eventTypes/update?batch=1");
+  await page.locator('[type="submit"]').click();
+  const response = await submitPromise;
+  expect(response.status()).toBe(200);
+}
```

Comment:
I found this pattern to be a reliable way to replace waiting on toast-success after saving changes. cc @anikdhabal

Reply:
I've noticed that sometimes a reload occurs after a successful save, especially for event types, and the toast message isn't visible. This is why expecting a success toast message can be flaky. I think we can use it in other tests and apply it to other event type update cases as well